std/file.d

```diff
@@ -886,11 +886,20 @@ version (Windows) private void writeImpl(scope const(char)[] name, scope const(F
 }
 
 /***************************************************
- * Rename file `from` _to `to`.
+ * Rename file `from` _to `to`, moving it between directories if required.
  * If the target file exists, it is overwritten.
+ *
+ * On Posix, the operation is atomic, which means, that it is not possible to
```

Comment:
POSIX is "POSIX", not "Posix". `Posix` is just what the `version` label is called for conformity with C.

Reply:
Most of the time, it's called "Posix" in the docs of this file.

Reply:
That should be fixed too.

Reply:
I'll do this in a sperate PR.